File: android_application/app/src/main/java/org/techtown/smartkey_app/add_category.java
package org.techtown.smartkey_app;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.Button;
import android.widget.Spinner;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class add_category extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_add_category);

        final TextView textView = (TextView)findViewById(R.id.textView);
        Spinner spinner = (Spinner)findViewById(R.id.spinner);

        spinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
            @Override
            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
                textView.setText(parent.getItemAtPosition(position) + " 이(가) 맞나요?");
            }
            @Override
            public void onNothingSelected(AdapterView<?> parent) {}
        });

        Button button1 = (Button) findViewById(R.id.exit_btn);
        button1.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(), lock.class);
                startActivity(intent);
            }
        });
    }
}

File: smartKey_App/app/src/main/java/com/example/smartkey_app/add_category.kt
package com.example.smartkey_app

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.AdapterView
import android.widget.AdapterView.OnItemSelectedListener
import android.widget.Spinner
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_add_category.*

class add_category : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_add_category)

        var textView = findViewById<View>(R.id.dddd) as TextView
        var spinner = findViewById<View>(R.id.spinner) as Spinner

        spinner.setOnItemSelectedListener(object : OnItemSelectedListener {
            override fun onItemSelected(
                parent: AdapterView<*>,
                view: View,
                position: Int,
                id: Long
            ) {
                textView.setText(parent.getItemAtPosition(position).toString() + " 이(가) 맞나요?")
            }

            override fun onNothingSelected(parent: AdapterView<*>?) {}
        })

        register_btn.setOnClickListener {
            var intent = Intent(applicationContext, lock::class.java)
            startActivity(intent)
        }
    }
}
